exchange/bitswap/workers.go

```diff
@@ -182,10 +182,11 @@ func (bs *Bitswap) rebroadcastWorker(parent context.Context) {
 	defer cancel()
 
 	broadcastSignal := time.After(rebroadcastDelay.Get())
+	tick := time.Tick(10 * time.Second)
```

Comment:
Today.




And not if we start many nodes and tear them down in one binary. Found a good bug. We should finish the fix. Do we lose anything by adding a stop?



—
Sent from Mailbox

On Sat, May 2, 2015 at 1:50 PM, Jeromy Johnson <notifications@github.com>
wrote:

>> @@ -182,10 +182,11 @@ func (bs *Bitswap) rebroadcastWorker(parent context.Context) {
>>  	defer cancel()
>>  
>>  	broadcastSignal := time.After(rebroadcastDelay.Get())
>> +	tick := time.Tick(10 * time.Second)
> Eh, not really needed. This is a single goroutine that lasts the entire lifetime of the executable. 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/ipfs/go-ipfs/pull/1180/files#r29550583